webservices/resources/legal.py

```diff
@@ -134,28 +134,66 @@ def get(self, q='', from_hit=0, hits_returned=20, type='all', **kwargs):
         return results
 
 def apply_mur_specific_query_params(query, **kwargs):
-    if kwargs.get('no'):
-        query = query.query('terms', no=kwargs.get('no'))
-    if kwargs.get('respondents'):
-        query = query.query('match', respondents=kwargs.get('respondents'))
-    if kwargs.get('election_cycles'):
-        query = query.query('term', election_cycles=kwargs.get('election_cycles'))
-    if kwargs.get('document_category') and kwargs.get('document_text'):
+    if kwargs.get('mur_no'):
+        query = query.query('terms', no=kwargs.get('mur_no'))
+    if kwargs.get('mur_respondents'):
+        query = query.query('match', respondents=kwargs.get('mur_respondents'))
+    if kwargs.get('mur_election_cycles'):
+        query = query.query('term', election_cycles=kwargs.get('mur_election_cycles'))
+    if kwargs.get('mur_document_category') and kwargs.get('mur_document_text'):
         combined_query = [
-            Q('match', documents__category=kwargs.get('document_category')),
-            Q('match', documents__text=kwargs.get('document_text'))]
+            Q('match', documents__category=kwargs.get('mur_document_category')),
+            Q('match', documents__text=kwargs.get('mur_document_text'))]
         query = query.query("nested", path="documents", query=Q('bool', must=combined_query))
 
     return query
 
 def apply_ao_specific_query_params(query, **kwargs):
-    query = query.query('match', category='Final Opinion')
+    categories = {'F': 'Final Opinion',
```

Comment:
@vrajmohan probably the way to do that would be to expose it in the api. Seems strange to have an endpoint for a static value though. I guess it would be good in case it ever changes though? Ugh, I don't know.